usr/src/uts/common/fs/zfs/dmu_send.c

```diff
@@ -1707,7 +1712,7 @@ dmu_recv_resume_begin_sync(void *arg, dmu_tx_t *tx)
 	dsobj = ds->ds_object;
 	dsl_dataset_rele(ds, FTAG);
 
-	VERIFY0(dsl_dataset_own_obj(dp, dsobj, dmu_recv_tag, &ds));
+	VERIFY0(dsl_dataset_own_obj(dp, dsobj, dmu_recv_tag, B_FALSE, &ds));
```

Comment:
ZOL only has one dsl_dataset_own_obj call in `dmu_send.c` and it looks to be because they do not have resume send in there. But presumably both `recv_begin_sync` and `recv_resume_begin_sync` should have the same value, so I have corrected it. Well spotted. I will push in a few, I want to compare the zol and illumos commits again, and see if I can find any other differences.